_posts/2022-09-29-r-text/r-text.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: "The text-package"
+description: |
+  The [text](http://r-text.org/)-package attempts to provide user-friendly access and pipelines to HuggingFace's transformer language models in R.
+author:
+  - name: Oscar Kjell 
+    url: https://oscarkjell.se/
+    affiliation: Lund University
+    affiliation_url: https://portal.research.lu.se/en/persons/oscar-kjell
+  - name: Salvatore Giorgi 
+    url: http://www.salvatoregiorgi.com/
+    affiliation: University of Pennsylvania
+    affiliation_url: http://www.salvatoregiorgi.com/
+  - name: H Andrew Schwartz  
+    url: https://www3.cs.stonybrook.edu/~has/
+    affiliation: Stony Brook University
+    affiliation_url: https://www3.cs.stonybrook.edu/~has/
+categories:
+  - Natural Language Processing
+  - Machine Learning
+  - Transformers
+  - HuggingFace
+date: 09-29-2022
+creative_commons: CC BY
+repository_url: https://github.com/OscarKjell/ai-blog
+output: distill::distill_article
+preview: img/ai_blog_word_plot.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE,
+                      eval = FALSE)
+```
+
+AI-based language analysis has recently gone through a “paradigm shift” (Bommasani, et al., 2021, p. 1) thanks in part to a new technique referred to as transformer language model (Vaswani et al., 2017, Liu et al., 2019). Companies, including Google, Meta, and OpenAI have released such models including BERT, RoBERTa, and GPT, that have achieved unprecedented large improvements across most language tasks such as web search and sentiment analysis. While these language models are accessible in Python, and for typical AI tasks through [HuggingFace](https://huggingface.co/), the R-package [text](http://r-text.org/) makes HuggingFace and state-of-the-art transformer language models accessible as social scientific pipelines in R.
```

Comment:
GH suggestions for some reason does not work here, so I'm entering things manually :-)

Bommasani, et al., -> Bommasani et al.,

thanks -> , thanks

including -> , including

R-package -> R package

Reply:
done